lib/routes/authorization.js

```diff
@@ -29,15 +35,30 @@ function set(arr) {
   return Object.keys(obj);
 }
 
-
+function scopeStringToSet(scopes) {
+  return typeof scopes === 'string' ?
+    set(scopes.split(' ')) :
```

Comment:
It may be worth doing `.split(/\s+/)` here to avoid weirdness if they accidentally pass multiple spaces in the string.